libopenage/coord/CMakeLists.txt

```diff
@@ -8,3 +10,8 @@ add_sources(libopenage
 	term.cpp
 	tile.cpp
 )
+
+target_include_directories(libopenage
+	PRIVATE
+		${CMAKE_CURRENT_BINARY_DIR}
```

Comment:
same here

Reply:
> The generated files cannot reference the header files using relative paths.

Conversely: the generated header files cannot be referenced using relative paths, either.

Reply:
mhm. damn.